backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ClientProvider.java

```diff
@@ -19,22 +19,29 @@
 package org.apache.james.backends.es;
```

Comment:
As a side note, loading self signed certificates, one way or another will need to be supported.